---
title: Curated Single Cell Multimodal Landmark Datasets for R/Bioconductor
date: '2021-01-01'
publishDate: '2024-12-27T20:18:58.633910Z'
authors:
- Kelly B. Eckenrode
- Dario Righelli
- Marcel Ramos
- Ricard Argelaguet
- Christophe Vanderaa
- Ludwig Geistlinger
- Aedin Culhane
- Laurent Gatto
- Vincent Carey
- Martin Morgan
- Davide Risso
- Levi Waldron
publication_types:
- '2'
abstract: The majority of high-throughput single-cell molecular profiling methods
  quantify RNA expression; however, recent multimodal profiling methods add simultaneous
  measurement of genomic, proteomic, epigenetic, and/or spatial information on the
  same cells. The development of new statistical and computational methods in Bioconductor
  for such data will be facilitated by easy availability of landmark datasets using
  standard data classes. We collected, processed, and packaged publicly available
  landmark datasets from important single-cell multimodal protocols, including CITE-Seq,
  ECCITE-Seq, SCoPE2, scNMT, 10X Multiome, seqFISH, and G&T. We integrate data modalities
  via the MultiAssayExperiment Bioconductor class, document and re-distribute datasets
  as the SingleCellMultiModal package in Bioconductor’s Cloud-based ExperimentHub.
  The result is single-command actualization of landmark datasets from seven single-cell
  multimodal data generation technologies, without need for further data processing
  or wrangling in order to analyze and develop methods within Bioconductor’s ecosystem
  of hundreds of packages for single-cell and multimodal data. We provide two examples
  of integrative analyses that are greatly simplified by SingleCellMultiModal. The
  package will facilitate development of bioinformatic and statistical methods in
  Bioconductor to meet the challenges of integrating molecular layers and analyzing
  phenotypic outputs including cell differentiation, activity, and disease.
featured: false
publication: '*bioRxiv*'
doi: 10.1101/2021.10.27.466079
---

